rpc/connection.go

```diff
@@ -331,6 +337,9 @@ type ConnectionOpts struct {
 	// before reconnecting. The random backoff timer is fast-forward-able by
 	// passing in a WithFireNow(ctx) into a RPC call.
 	InitialReconnectBackoffWindow time.Duration
+	// DialerTimeout is the Timeout used in net.Dialer when initiating new
+	// connections.
```

Comment:
Mention that a zero value means no timeout?

Reply:
Sure will do!